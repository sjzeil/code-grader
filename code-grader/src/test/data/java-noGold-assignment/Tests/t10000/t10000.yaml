onFail: ["t10000_failed"]
