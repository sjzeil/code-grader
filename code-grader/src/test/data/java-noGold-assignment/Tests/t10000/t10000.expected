The square root of 10000.00 is 100.0000.
